src/sys/windows/selector.rs

```diff
@@ -626,7 +585,7 @@ impl SelectorInner {
                 n += 1;
                 continue;
             }
-            let sock_arc = Arc::from_raw(iocp_event.overlapped() as *const Mutex<SockState>);
+            let sock_arc: Pin<Arc<Mutex<SockState>>> = transmute_copy(&iocp_event.overlapped());
```

Comment:
is this a perfect copy without increasing the ref count?

Reply:
Yep. It's basically reinterpret_cast + copy.